SConstruct

```diff
@@ -30,7 +30,7 @@ def validate(cls, root, dataId=None, **kwargs):
         assert len(kwargs) == 0  # There's no dataId to update
     cmd = [getExecutable("ci_hsc", "validate.py"), cls.__name__, root,]
     if dataId:
-        cmd += ["--id %s" % (" ".join("%s=%s" % (key, value) for key, value in dataId.iteritems()))]
+        cmd += ["--id %s" % (" ".join("%s=%s" % (key, value) for key, value in iter(dataId.items())))]
```

Comment:
Do you need the `iter()` at all? I'm assuming there aren't millions of items so using a list on py2 rather than an iterator shouldn't damage performance.